README.md

```diff
@@ -49,7 +49,7 @@ You can see a (customized) instance of Strapping in action by visiting http://oV
 2. Clone the repository:
 
    ```
-   git clone https://github.com/OSAS/strapping-mediawiki strapping
+   git clone https://github.com/axxie/strapping-mediawiki strapping
```

Comment:
axxie:
> Looks like original author, @garrett, abandoned the project long time ago.

I am wondering about that also. Not sure. They are still using it:

- http://old.ovirt.org/Home
- http://www.ovirt.org/

Perhaps he is busy. Perhaps it works good enough for him so he only
reacts to and merges stuff that is of personal interest to him.

Do you plan to fork and maintain strapping-mediawiki? @axxie